app/src/main/java/com/spendster/presentation/validation/ComplexUsernameValidator.java

```diff
@@ -0,0 +1,20 @@
+package com.spendster.presentation.validation;
+
+public class ComplexUsernameValidator implements Validation<ValidationResource> {
```

Comment:
@SimonMaha It's really hard to call the validation in this class a complex one :)